src/ts/components/Match.tsx

```diff
@@ -32,6 +38,14 @@ class Match<TMatch extends IMatch> extends Component<IMatchProps<TMatch>> {
                 matchId={matchId}
                 suggestions={suggestionsToRender}
               />
+              <div className="MatchWidget__feedbackLink">
+                <a 
```

Comment:
This div is within `MatchWidget__suggestion-list` which is not meant to display when there are no suggestions. I recommend you move it out.

Reply:
Done.